DigitalLearningSolutions.Web/ViewModels/LearningMenu/DiagnosticContentViewModel.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
+{
+    using System.Collections.Generic;
+    using System.Text.RegularExpressions;
+    using DigitalLearningSolutions.Data.Models.DiagnosticAssessment;
+    using DigitalLearningSolutions.Web.Helpers;
+    using Microsoft.Extensions.Configuration;
+
+    public class DiagnosticContentViewModel
+    {
+        public int CustomisationId { get; }
+        public int SectionId { get; }
+        public string CourseTitle { get; }
+        public string SectionName { get; }
+        public string ContentSource { get; }
+        public List<int> Tutorials { get; }
+        private static readonly Regex ScormRegex = new Regex(@".*imsmanifest\.xml$");
+        private static bool IsScormPath(string path) => ScormRegex.IsMatch(path);
```

Comment:
Don't we have this in another class as well? If so, probably worth extracting so it can be shared 🙂